src/components/BatchVoting/DefaultOptions/DefaultOptions.tsx

```diff
@@ -0,0 +1,132 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React, { FC, useEffect, useState } from 'react';
+import OptionWrapper from './OptionWrapper';
+import { useNetworkSelector, useUserDetailsSelector } from '~src/redux/selectors';
+import { ILastVote, Wallet } from '~src/types';
+import Alert from '~src/basic-components/Alert';
+import { ProposalType } from '~src/global/proposalType';
+import CustomButton from '~src/basic-components/buttons/CustomButton';
+import { useAppDispatch } from '~src/redux/store';
+import { batchVotesActions } from '~src/redux/batchVoting';
+import AddressDropdown from '~src/ui-components/AddressDropdown';
+import getAccountsFromWallet from '~src/util/getAccountsFromWallet';
+import { InjectedAccount, InjectedWindow } from '@polkadot/extension-inject/types';
+import { isWeb3Injected } from '@polkadot/extension-dapp';
+import { useApiContext } from '~src/context';
+import { IDefaultOptions } from '../types';
+
+const DefaultOptions: FC<IDefaultOptions> = (props) => {
+	const { forSpecificPost, postEdit } = props;
```

Comment:
no need to assign to intermediate variable

Reply:
resolved